config.ru

```diff
@@ -1,3 +1,17 @@
 # config.ru
+require 'rack/env'
+require_relative './config/environment'
+
+if ENV['RACK_ENV']=='production'
+  redis_hash = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}", password: "#{ENV['REDIS_PASSWORD']}" }
```

Comment:
you should put this line after load `Rack::Env`

Reply:
```suggestion
use Rack::Env
  redis_hash = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}", password: "#{ENV['REDIS_PASSWORD']}" }
```